### DISCUSSION TITLE
Setting management UI

### DISCUSSION THREAD

**Discussion by emilsjog** [abpframework/abp#9315] [Q&A]
Hi,

Is there any documentation or example project on how to use/extend the setting management UI with custom settings? I have tried the emailing settings that are included in the startup template. And I noticed that the settings are only stored in the DB after editing them.

I have added some settings to the appsettings.json file accoring to the [documentation](https://docs.abp.io/en/abp/latest/Settings) and can access them with the `ISettingProvider`. But what is the best practice to store them in the DB before editing them? Is it to seed them in?

ABP version 4.3.2

**Comment by maliming** [comment]
You can refer to this pr https://github.com/abpframework/abp/pull/7669

**Reply by emilsjog** [reply]
THanks, @maliming 
Got it working and added another settings group. 
![image](https://user-images.githubusercontent.com/6585041/122042094-011a5680-cdda-11eb-9c3b-56d23650d78b.png)

But I cant get permissions for only one group to work. I have to enable permissions for **Emailing** otherwise there is no access to the whole page. Any tips?

**Reply by maliming** [reply]
Please waitting for 4.3.3

https://github.com/abpframework/abp/pull/9213
